frontend/src/components/GoodBye/GoodbyeMessage/GoodbyeMessage.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter, useNavigate } from 'react-router-dom';
+import { describe, expect, it, Mock, vi, beforeEach } from 'vitest';
+import userEvent from '@testing-library/user-event';
+import GoodByeMessage from './GoodbyeMessage';
+
+vi.mock('react-router-dom', async () => {
+  const mod = await vi.importActual<typeof import('react-router-dom')>('react-router-dom');
+  return {
+    ...mod,
+    useNavigate: vi.fn(),
+  };
+});
+
+const mockNavigate = vi.fn();
+
+describe('GoodbyeMessage', () => {
+  const headerMessage = 'This is a header message';
+  const goodbyeMessage = 'This is a header message';
```

Comment:
Nit: maybe have different messages?

Reply:
ahh, just a stray copy-pasta. good catch, thanks, I updated it! 🚀 🙏